src/lib/y2network/connection_config/base.rb

```diff
@@ -166,6 +175,13 @@ def find_master(configs)
         end
       end
 
+      # Returns whether the connection is configured as static
+      #
+      # @return [Boolean] return true if the connection is static; false otherwise
+      def static?
+        bootproto == BootProtocol::STATIC
```

Comment:
I am not sure if it is good place here as connection config quite start growing. What about doing it same as I did it for interface type and add helpers to BootProtocol? so it can be `bootproto.static?`